app/src/main/java/com/kabouzeid/gramophone/ui/fragments/player/PlayerAlbumCoverFragment.java

```diff
@@ -68,13 +83,16 @@ public boolean onTouch(View v, MotionEvent event) {
                 return gestureDetector.onTouchEvent(event);
             }
         });
+        progressViewUpdateHelper = new MusicProgressViewUpdateHelper(this, 500, 1000);
+        progressViewUpdateHelper.start();
     }
 
     @Override
     public void onDestroyView() {
         super.onDestroyView();
         viewPager.removeOnPageChangeListener(this);
         unbinder.unbind();
+        progressViewUpdateHelper.stop();
```

Comment:
Should probably stop it before unbinding, just in case something weird happens and it tries to update it right after the view's unbound.